### PR TITLE
Responsive

### PR DESCRIPTION
**Pull request by obaidofj**
responsive design
as this -normal size/big :
etch6
 smaller size:
etch7
also same as for smaller sizes are responsive

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+drawGrid(gridsize); //grid by default size
+
+document.getElementById("newGridBut").addEventListener("click", function () {
+  //new grid listener
+  let validNumber = false,
+    newGridSize = 0;
+  do {
+    newGridSize = Number(prompt("What size of the grid you want ?", 16));
+    if (isNaN(newGridSize)) {
+      alert("Enter valid number");
```
Comment (Jood80) [inline script.js:96]: you need also to validate if the user enters a space or hit **OK** after writing nothing using **.trim()** method for instance- you can add it in the same if statement here-. Also, you need to consider  entering **"0"** case validation

File: style.css [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+  border: 1px solid rgb(59, 59, 59);
+  background: #efefef;
+  padding: 5px;
+  margin-top: 4px;
+  display: inline-block;
+  font-size: 15px;
+}
+/*input.chk-btn:not(:checked) + label:hover {
+  box-shadow: 0px 1px 3px;
+}*/
```
Comment (Jood80) [inline style.css]: for a clean code, any unused code or such comment has to be removed

**Review by Jood80 — CHANGES_REQUESTED** [review]
You nailed it! :v: :tada: 
I left a few things to be updated, otherwise GREAT work!

File: script.js (hunk 1 above)
Comment (obaidofj) [reply script.js:96]: ok in make the changes to handle that ..

File: style.css (hunk 2 above)
Comment (obaidofj) [reply style.css]: ok i removed it

**Review by Jood80 — APPROVED** [review]
GREAT JOB :100:

File: index.html [hunk 3]
```diff
@@ -10,18 +10,22 @@
...
-    <input type="radio" name="mode" id="radioBut3PickColor" value="pikClr">Pick Color<input type="color" id="clr"></input></input>
+       <br> &nbsp;&nbsp;&nbsp;  <strong>Mode: </strong> &nbsp;&nbsp; 
+    <input type="radio" name="mode" id="radioBut1RandomColors" value="randomClrs" checked="checked"><label for="radioBut1RandomColors">&nbsp;Random Colors</label> </input>
+    <span class="spaces">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span>&nbsp;&nbsp;&nbsp;<div id="together"><input type="radio" name="mode" id="radioBut2RandomColor" value="randomClr"><label for="radioBut2RandomColor">&nbsp;Random Color</label>  <div id="clrSelected"></div></input></div>
+    <span class="spaces">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span>&nbsp;&nbsp;&nbsp;<div id="together"><input type="radio" name="mode" id="radioBut3PickColor" value="pikClr"><label for="radioBut3PickColor">&nbsp;Pick Color</label></input>&nbsp;<input type="color" id="clr"></input></div>
   </div>
```
Comment (Jood80) [inline index.html]: One last thing which is **dangerous** to keep in your code, each element must has a unique Id, you named three divs with the same <id="together"/> which is affect your accessibility as well. 
PS: changing the [id] is amust. but adding the label for your form is up to you to increase your score of accessibility.
image

**Review by Jood80 — CHANGES_REQUESTED** [review]

**Review by Rahaf-96 — APPROVED** [review]
you did an amazing job! :tada:

**Review by Jood80 — APPROVED** [review]
WELL DONEEE :tada:

**Review by saharAdem — APPROVED** [review]

**Pull request merged**